src/Dialect/ONNX/ONNXOps/Math/ElementwiseUnary.cpp

```diff
@@ -257,6 +257,28 @@ LogicalResult ONNXFloorOp::inferShapes(
   return inferShapeForUnaryOps(this->getOperation());
 }
 
+//===----------------------------------------------------------------------===//
+// Gelu
+//===----------------------------------------------------------------------===//
+LogicalResult ONNXGeluOp::verify() {
```

Comment:
Thanks for the verifier, very complete implementation, appreciated.

Reply:
You are welcome!!